hooks/pre-push.d/toml-license-field-test

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+files_without_license=$(find . -path ./target -prune -o -type f -name "Cargo.toml" -exec grep -l "\[package\]" {} \; | xargs grep -L "license = \"Apache-2.0\"")
+[ -z "$files_without_license" ] && exit 0
+printf "The following Cargo.toml files are missing Apache 2.0 license fields:\n$files_without_license\n" && exit 1
```

Comment:
I think we need to properly parse the toml here. This test would pass for a toml that looks like this:

```toml
license = "Apache-2.0"
[package]
```

Reply:
This still needs to be done.